### ISSUE TITLE
Licensing

### ISSUE THREAD

**Issue by sigmavirus24** [sigmavirus24/python-interview-questions#2]
So without a license, no one can redistribute this anywhere. Do we want to add a license? If so which do we want to add?

Do we want to add solutions to these questions? If so, do we care about the license of that code? Do we only care about the markdown questions and receiving attribution? Are there other things to consider?

**Comment by szeitlin** [comment]
coming from academia, I'd say let's include links to answers, if we've solved the questions ourselves or found good solutions, and use whatever license says 'can use as long as attributed'. What is that, Creative Commons? I'm relatively inexperienced with licensing.

**Comment by sigmavirus24** [comment]
Yep! So if you look at https://creativecommons.org/choose/ there are some options. I'm leaning towards a certain set of options, but I'd love your feedback =D

**Comment by malaman** [comment]
I think MIT License (MIT) is suitable for the project.

**Comment by sigmavirus24** [comment]
## Can you explain why? 

Sent from my Android device with K-9 Mail. Please excuse my brevity.

**Comment by malaman** [comment]
I thought about MIT license as I treated the project as software project (maybe some code will be added to the project in future)

As far as i know CC licenses aren't meant to software projects (https://wiki.creativecommons.org/Frequently_Asked_Questions#Can_I_apply_a_Creative_Commons_license_to_software.3F)

But if the project remains list of questions, CC license is suitable for it.

**Comment by szeitlin** [comment]
Not sure how this differs from creative commons: http://www.apache.org/licenses/LICENSE-2.0

**Comment by malaman** [comment]
Agreed. CC is suitable.

**Comment by webmaven** [comment]
@szeitlin , the Apache license is just about as permissive, and includes language covering patents, but like MIT, isn't really appropriate for prose.

I would generally suggest the CC-BY license if you want something permissive, or the CC-BY-SA license if you want something copyleft.
